src/transactions/AllowTrustOpFrame.cpp

```diff
@@ -16,6 +16,30 @@
 
 namespace stellar
 {
+
+void
```

Comment:
For future reference: it's generally advisable to move code, then edit it. This is going to be annoying when you address my comments from the previous commit because it will produce conflicts with this.

Reply:
Good point. I'll keep that in mind.